src/modules/acc/doc/acc_admin.xml

```diff
@@ -1290,7 +1290,7 @@ modparam("acc", "cdrs_table", "acc_cdrs")
 				microseconds in time_exten.</para>
 		</listitem>
 		<listitem>
-			<para><emphasis>2</emphasis> - save seconds.milliseconds
+			<para><emphasis>2</emphasis> - save seconds.microseconds
```

Comment:
Shouldn't be better a new `time_mode` value (e.g., 5) for this, instead of changing the behaviour for 2? Existing deployments might be affected, requiring probably external billing apps to be upgraded as well.

Reply:
I did consider a different mode but the DB structure would need changed in either case so I did not think there was a good way to maintain complete backwards compatibility.  
That was my intial thought but as I was thinking this over last night I realized we can gracefully handle the check for that change in schema in the module initialization.  
@miconda I will reformat this PR and tag as WIP until those changes are in